jcs/openst.py

```diff
@@ -41,7 +41,7 @@ def os_instance_create(self, instance_name, instance_type, image_name, key_name,
         # create instance
         instance = self._conn.create_server(
             instance_name, image=image, flavor=instance_type, key_name=key_name,
-            network=network_fixed, security_groups=security_groups,
+            network=network_public, security_groups=security_groups,
```

Comment:
I suppose the reason it was working before, because while it was trying to the fixed network in the line 39 it was return Null value and passed as a value for parameter _network_, and because there was just one public network it was just taking it. Now there is more than one network on some region, so it started to fail.
This change is fine, but it make useless the line 39, and the parameter *network_fixed* in this method. So I suggest you to take the fixed network into account, for example, providing parameter like `network=network_fixed or network_public`

Reply:
Makes sense. Using `network_public` only in case `network_fixed` is a `null` value is indeed less risky wrt. backwards compatibility.